smarts/core/mission_planner.py

```diff
@@ -202,44 +202,48 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
         return edge_ids
 
     def cut_in_waypoints(self, sim, pose: Pose, vehicle):
-        radius = self._mission.task.trigger_radius
+
+        aggressiveness = 5
         neighborhood_vehicles = sim.neighborhood_vehicles_around_vehicle(
-            vehicle=vehicle, radius=radius
+            vehicle=vehicle, radius=80
         )
 
         position = pose.position[:2]
         lane = self._road_network.nearest_lane(position)
 
-        if neighborhood_vehicles:
-            nei_vehicle = neighborhood_vehicles[0]
+        if not neighborhood_vehicles or sim.elapsed_sim_time < 3:
```

Comment:
Sorry, I think 1 second is more reasonable after the test.

Reply:
No worries, sure.